### DISCUSSION TITLE
Failed to send test ETH from Ethereum Sepolia to my account

### DISCUSSION THREAD

**Discussion by yangpeng216** [smartcontractkit/Web3_tutorial_Chinese#30] [Q&A] [answered]
I failed to send any test ETH from Ethereum Sepolia to my account.  I have tried it many times. The attachment is the error screenshot.
<img width="933" alt="image" src="https://github.com/user-attachments/assets/99c79a6a-8f96-47eb-8794-8d2fba017f85" />

**Comment by QingyangKong** [comment, marked as answer]
需要你在主网的相同地址有一个 LINK 才可以。

**Reply by lingjs90** [reply]
请问怎么在主网地址有1个link？

**Reply by QingyangKong** [reply]
需要在交易所购买一个 LINK，然后转到你的地址上
